### PR TITLE
Create reverse_number.c

### PR DESCRIPTION
**Pull request by Vaibhav1671998**
Created a file reverse a number reverse_number.c

### CODE REVIEW

File: code/mathematical_algorithms/reverse_number/reverse_number.c [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#include<stdio.h>
+
+int reverse_number(int n){
+	int ans = 0;
+	while(n != 0){
+		ans *= 10;
+		ans += n % 10;
+		n /= 10;
+	}
+    return ans;
```
Comment (iattempt) [inline code/mathematical_algorithms/reverse_number/reverse_number.c]: Indentation is hard tab.

File: code/mathematical_algorithms/reverse_number/reverse_number.c [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#include<stdio.h>
+
+int reverse_number(int n){
+	int ans = 0;
+	while(n != 0){
```
Comment (iattempt) [inline code/mathematical_algorithms/reverse_number/reverse_number.c]: Space should be followed after condition keyword

**Comment by Vaibhav1671998** [comment]
@iattempt Done

**Comment by iattempt** [comment]
please check coding style again

**Comment by Vaibhav1671998** [comment]
@iattempt Check it now

**Comment by Vaibhav1671998** [comment]
@arnavb Have a look at this PR .

File: code/mathematical_algorithms/reverse_number/reverse_number.c [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+		n /= 10;
+	}
+	return ans;
+}
+
+int main(){
+	int n;
+	printf("Enter the number : ");
+	scanf("%d",&n );
+	printf("Reverse of %d is %d",n,reverse_number(n));
```
Comment (arnavb) [inline code/mathematical_algorithms/reverse_number/reverse_number.c]: Add a space after each comma

File: code/mathematical_algorithms/reverse_number/reverse_number.c [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+		ans += n % 10;
+		n /= 10;
+	}
+	return ans;
+}
+
+int main(){
+	int n;
+	printf("Enter the number : ");
+	scanf("%d",&n );
```
Comment (arnavb) [inline code/mathematical_algorithms/reverse_number/reverse_number.c]: Remove the space before the closing parenthesis, and add a space after the comma.

File: code/mathematical_algorithms/reverse_number/reverse_number.c [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+		ans *= 10;
+		ans += n % 10;
+		n /= 10;
+	}
+	return ans;
+}
+
+int main(){
+	int n;
+	printf("Enter the number : ");
```
Comment (arnavb) [inline code/mathematical_algorithms/reverse_number/reverse_number.c]: Enter a number to reverse:

File: code/mathematical_algorithms/reverse_number/reverse_number.c [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+	while (n != 0){
+		ans *= 10;
+		ans += n % 10;
+		n /= 10;
+	}
+	return ans;
+}
+
+int main(){
+	int n;
```
Comment (arnavb) [inline code/mathematical_algorithms/reverse_number/reverse_number.c]: Default initialize

File: code/mathematical_algorithms/reverse_number/reverse_number.c [hunk 7]
```diff
@@ -0,0 +1,20 @@
+#include<stdio.h>
+
+int reverse_number(int n){
```
Comment (arnavb) [inline code/mathematical_algorithms/reverse_number/reverse_number.c]: I believe you should be using camelCase, but first check the Style Guide.

File: code/mathematical_algorithms/reverse_number/reverse_number.c [hunk 8]
```diff
@@ -0,0 +1,20 @@
+#include<stdio.h>
```
Comment (arnavb) [inline code/mathematical_algorithms/reverse_number/reverse_number.c]: Add space after include

**Comment by Vaibhav1671998** [comment]
@arnavb I had done the changes .
Again i don't think default initialization is necessary as i am taking the input .
Rest changes are DONE :100:

**Comment by Vaibhav1671998** [comment]
@AdiChat Merge this is u r happy :)

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**